_client/src/Dialogs/RegistrationDialog.cpp

```diff
@@ -52,9 +52,31 @@ void RegistrationDialog::setupLayout()
     SPDLOG_TRACE("RegistrationDialog::setupLayout");
 
     auto *mainLayout = new QVBoxLayout(this);
-    mainLayout->addWidget(m_usernameField);
-    mainLayout->addWidget(m_passwordField);
-    mainLayout->addWidget(m_confirmPasswordField);
+
+    ///> Layout for labels
+    std::vector<QWidget *> labels;
+    labels.push_back(new QLabel(tr("Username")));
+    labels.push_back(new QLabel(tr("Password")));
+    labels.push_back(new QLabel(tr("Confirm password")));
+
+    auto *labelLayout = Utils::createLayout(labels);
```

Comment:
You need to remember, that Qt handles dynamic object memory allocation only for child QObjects.
Here you just create labelLayout, so you need either manage deletion of the object by yourself (`delete labelLayout` in DialogManager destructor) or use this pointer when creating object, so DialogManager will manage this for you.

Reply:
fix